design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -0,0 +1,261 @@
+## Gift Card product type support will be added after the Product Implementation is developed
+type Query {
```

Comment:
Would be nice to have sequence of queries/mutations for the specific scenarios documented. That would help to ensure completeness and compatibility of all operations. Additionally that would serve as a good starting point for new automated tests.